storage/bootstrap/types.go

```diff
@@ -62,6 +75,23 @@ type RunOptions interface {
 	// Incremental returns whether this bootstrap should be an incremental
 	// that saves intermediate results to durable storage or not.
 	Incremental() bool
+
+	// SetIndexingEnabled sets whether indexing is enabled which determines
```

Comment:
I think you can drop these methods from the `RunOptions()`. `namespaceMetadata.Options().IndexOptions().Enabled()` has this flag now.

Reply:
Ok sure thing, sounds good.